content/blog/2023/07/10/2023-07-10-jenkins-june-newsletter.adoc

```diff
@@ -0,0 +1,161 @@
+
+[[security-fixes]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/security.png[Security Update]
+
+Contributed by: author:wadeck[Wadeck Follonier]
+
+* Particular security advisory published on June 14
+** link:/security/advisory/2023-06-14/[link:security/advisory/2023-06-14/]
+** The security team discovered a vulnerability that was corrected as a (positive) side effect of a maintenance task.
+So new security release per se for Jenkins Core, but still an advisory and associated warnings to let administrators know that an update is recommended.
+** That advisory also included fixes for plugins.
+* Expansion of the security audit scope in Jenkins Core
+** Originally the scope of the audit requirement was only “UI-related” changes.
+** But recently there were multiple changes in JavaScript / Jelly without UI impact, mainly for maintenance.
+** More information about this is in the link:https://docs.google.com/document/d/1QttPwdimNP_120JukigKsRuBvMr34KZhVfsbgq1HFLM[Jenkins UX SIG - Agendas & Notes], for June 21.
+* Special thanks to the GitHub Security Lab for their reports
+** Second time in a row that vulnerabilities they reported were corrected in the advisory.
+** So, kudos to Alvaro Muñoz and Tony Torralba
+** In collaboration with them, the Jenkins Security team will improve the link:blog/2020/11/04/codeql/[CodeQL scanning tool].
+
+
+[[Governance]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/governance.png[Governance Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+The Linux Foundation will upgrade link:https://issues.jenkins.io[issues.jenkins.io] on Thursday, July 6, 2023. Special thanks to the Linux Foundation for their skilled administration of our JIRA instance.
+More detailed information is available on the link:https://status.jenkins.io/issues/2023-07-06-jira-outage/[status page].
+
+The four link:/projects/gsoc/#gsoc-2023[Google Summer of Code projects] mentored by members of the Jenkins project are preparing midterm presentations and midterm evaluations.
+The link:https://www.meetup.com/jenkins-online-meetup/events/294355266/[midterm webinar] will be Thursday, July 6, 2023. A recording will be available as well along with the link:https://docs.google.com/presentation/d/1kfGd0IB2PWp_yzSDFk5ClY00qZGreGjirtqL7-SZ1js/edit?usp=sharing[presentation slides].
+
+Upgrades and improvements of Jenkins components continue, with significant progress towards the eventual removal of Prototype.js from Jenkins core. We’ve also seen upgrades for Guava, Guice, Apache commons.io, and HTMLUnit. Thanks to the providers of those libraries and special thanks to the Jenkins maintainers that are leading those upgrades.
+
+Thanks to a kind donation from Launchable, pull requests to Jenkins core now complete their evaluation builds in 2 hours rather than the 6 hours that were previously required.
```

Comment:
```suggestion
Thanks to a kind donation from https://www.launchableinc.com/[Launchable], pull requests to Jenkins core now complete their evaluation builds in 2 hours rather than the 6 hours that were previously required.
```